src/pages/saksbehandling/søknadsbehandling/fast-opphold-i-norge/FastOppholdINorge.tsx

```diff
@@ -1,23 +1,23 @@
 import * as RemoteData from '@devexperts/remote-data-ts';
-import { useFormik } from 'formik';
+import { yupResolver } from '@hookform/resolvers/yup';
 import { Eq } from 'fp-ts/lib/Eq';
 import AlertStripe from 'nav-frontend-alertstriper';
 import { Feiloppsummering, Radio, RadioGruppe, Textarea } from 'nav-frontend-skjema';
 import NavFrontendSpinner from 'nav-frontend-spinner';
-import React, { useState } from 'react';
+import React, { useRef } from 'react';
+import { Controller, useForm } from 'react-hook-form';
 import { useHistory } from 'react-router-dom';
 
+import { FastOppholdFaktablokk } from '~components/oppsummering/vilkårsOppsummering/faktablokk/faktablokker/FastOppholdFaktablokk';
 import ToKolonner from '~components/toKolonner/ToKolonner';
-import { lagreBehandlingsinformasjon } from '~features/saksoversikt/sak.slice';
+import * as sakSlice from '~features/saksoversikt/sak.slice';
```

Comment:
I hvilke tilfeller bruker vi star-imports på egen kode? Og når ikke?

Reply:
Bruker star-imports når vi vil ha tak i et objekt med alle named-exportene fra filen. Dette er typisk for utility-moduler. Ellers så bruker vi default import.

Reply:
Men her ønsker vi kun lagreBehandlingsinformasjon? Er det fordi vi har navnekonflikt?

Reply:
Jepp. Jeg syns dette er en ryddig måte å gjøre det på (er generelt ikke så glad i unqualified imports), sånn at man nede i koden ser forskjell på hva som kaller til en ekstern modul (`sakSlice.lagreBehandlingsinformasjon`) og det som er en lokal hjelpefunksjon (`lagreBehandlingsinformasjon`).